src/WebJobs.Extensions.DurableTask/ContextImplementations/DurableOrchestrationClient.cs

```diff
@@ -153,6 +153,69 @@ Task IDurableOrchestrationClient.RaiseEventAsync(string taskHubName, string inst
             return this.RaiseEventInternalAsync(taskHubClient, taskHubName, instanceId, eventName, eventData);
         }
 
+        /// <inheritdoc />
+        Task IDurableOrchestrationClient.SignalActor(string taskHubName, ActorId actorId, string operationName, object operationContent, string connectionName)
+        {
+            if (string.IsNullOrEmpty(connectionName))
+            {
+                connectionName = this.attribute.ConnectionName;
+            }
+
+            var attribute = new OrchestrationClientAttribute
+            {
+                TaskHub = taskHubName,
+                ConnectionName = connectionName,
+            };
+
+            TaskHubClient taskHubClient = ((DurableOrchestrationClient)this.config.GetClient(attribute)).client;
+            return this.SignalActor(taskHubClient, taskHubName, actorId, operationName, operationContent);
+        }
+
+        /// <inheritdoc />
+        Task IDurableOrchestrationClient.SignalActor(ActorId actorId, string operationName, object operationContent)
+        {
+            if (string.IsNullOrEmpty(operationName))
+            {
+                throw new ArgumentNullException(nameof(operationName));
+            }
+
+            return this.SignalActor(this.client, this.hubName, actorId, operationName, operationContent);
+        }
+
+        private async Task SignalActor(TaskHubClient client, string hubName, ActorId actorId, string operationName, object operationContent)
+        {
+            if (string.IsNullOrEmpty(operationName))
+            {
+                throw new ArgumentNullException(nameof(operationName));
+            }
+
+            var guid = Guid.NewGuid(); // unique id for this request
+            var instanceId = ActorId.GetSchedulerIdFromActorId(actorId);
+            var instance = new OrchestrationInstance() { InstanceId = instanceId };
+            var request = new RequestMessage()
+            {
+                ParentInstanceId = null,
+                Id = guid,
+                IsSignal = true,
+                Operation = operationName,
+            };
+            if (operationContent != null)
+            {
+                request.SetContent(operationContent);
+            }
+
+            var jrequest = JToken.FromObject(request, MessagePayloadDataConverter.DefaultSerializer);
+            await client.RaiseEventAsync(instance, "op", jrequest);
+
+            this.traceHelper.FunctionScheduled(
+                hubName,
+                actorId.ActorClass,
+                ActorId.GetSchedulerIdFromActorId(actorId),
```

Comment:
Yes, we sort of do want that in the sense that it would be cleaner... I did that first but then I backed it out because it breaks a lot of the unit tests that do a full comparison of (expected trace) vs. (actual trace). Still, we can do it if you think it's worth it.